### ISSUE TITLE
Trello removed from modules

### ISSUE THREAD

**Issue by panckreous** [wtfutil/wtf#1068]
## How can we help?

Hi, it looks like commit 3c63eee accidentally deleted trello along with the todo directory. I didn't see it mentioned anywhere so I'm assuming it was a mistake (also, it's still in the docs), but posting this as a question instead of a PR just in case.

**Comment by senorprogrammer** [comment]
The Todo-like widgets have all been united under a common framework. Trello and Todo both still work and are configurable, they just no longer have their own directories. I should remove the source links from those pages, thanks for pointing that out.

**Comment by panckreous** [comment]
Ah, ya I saw the couple of todo widgets merge into that but didn't realize Trello counted as one too. (Of course it wasn't until right now that I saw `func FromTrello`) Makes sense. I just discovered this wonderful project a few hours ago and am less than fluent in Go, else I'd have seen and sent a PR with updated docs myself. 

Once I get Trello and a todo app working + spend a little more time overall, I'll see if I can find any places in the docs that would have helped me initially then submit a PR to hopefully help out there. 

Thanks again
